lib/node_modules/@stdlib/math/base/special/expm1rel/benchmark/python/scipy/benchmark.py

```diff
@@ -104,9 +104,9 @@ def main():
     """Run the benchmarks."""
     print_version()
 
-    name = ":TODO"
-    setup = "from scipy import TODO; from random import random;"
-    stmt = "y = TODO(random())"
+    name = "exprel"
```

Comment:
This can be an empty string. As indicated at L95, `name` is used for "sub-namespacing" of benchmarks. Here, that is not necessary.

If let as is, the benchmark name would be

```
# python::scipy::expm1rel:exprel
```
which is both redundant and confusing.

Reply:
Admittedly, we are missing documentation on our benchmark naming conventions, which is on our TODO list.

Reply:
Do you want it to read 
```
# python::scipy::special::exprel
```  
The scipy function I used as a template is exprel from the scipy.special module

Reply:
Yes, that is correct. Hence, if we set `name = ""`, then the benchmark name as printed in the terminal will be

```
# python::scipy::expm1rel:exprel
```